src/content/app/genome-browser/components/browser-sidebar-modal/services/browser-bookmarks-storage-service.ts

```diff
@@ -18,14 +18,14 @@ import storageService, {
   StorageServiceInterface
 } from 'src/services/storage-service';
 
-import { PreviouslyViewedObject } from 'src/content/app/genome-browser/state/track-panel/trackPanelSlice';
+import type { PreviouslyViewedObject } from 'src/content/app/genome-browser/state/browser-bookmarks/browserBookmarksSlice';
 
 export enum StorageKeys {
-  BOOKMARKS = 'trackPanel.bookmarks',
-  PREVIOUSLY_VIEWED = 'trackPanel.previouslyViewedObjects'
+  BOOKMARKS = 'browserSidebarModal.bookmarks',
+  PREVIOUSLY_VIEWED = 'browserSidebarModal.previouslyViewedObjects'
```

Comment:
storage key: replace the part `browserSidebarModal` with `genomeBrowser`.

Reply:
✅